HHS/OPRE-OPS
# Ops 1429/bug fix_round_two

## What changed

brings back action icons

## Issue

#1429 

## How to test

1. goto budgetlines tab
2. hover over row
3. see icons when hovering over status tag
